community-spotlight-conference-backend-world-affairs-conference.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Modernizing the conference management backend of the World Affairs Conference'
+description: 'Putting the spotlight on Samuel Martineau, a software engineer based out of Ontario, Canada replacing the backend of a conference management system with Xata.'
```

Comment:
In Canada, it is actually illegal to call yourself an engineer without proper degree or certification. I think that Apprentice Software Developer would be more appropriate.

Reply:
Very interesting! Updated ✅

Reply:
Updated ✅